app/containers/Recipes.js

```diff
@@ -0,0 +1,86 @@
+const React = require('react');
+const _ = require('lodash/core');
+const defaultRecipes = require('../assets/defaultRecipes.js');
+const RecipesHeading = require('../components/RecipesHeading');
+const RecipeList = require('../components/RecipeList');
+const RecipeFooter = require('../components/RecipeFooter');
+
+const recipeListingStyles = require('../styles/styles.js').recipeListingStyles;
+const listingFigTitleStyles = require('../styles/styles.js').listingFigTitleStyles;
+const listingFigTextStyles = require('../styles/styles.js').listingFigTextStyles;
+const listingDivStyles = require('../styles/styles.js').listingDivStyles;
+
+module.exports = class Recipes extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			recipes: [],
+			localStorageSupport: false
+		};
+		const self = this;
+
+		self.getLocalStorage = () => {
```

Comment:
I would try to abstract out the `get`ing of data from localstorage to the `setState` that affects this component. Similar to splitting the db connections/query with the sending db data to another place in code.